client/src/components/Notices/NoticeList.js

```diff
@@ -67,16 +66,32 @@ const NoticeList = ({
           <FormText>
             {searchValue && `${resultPlurality(noticesLength)} pre "${searchValue}".`}
           </FormText>
-          {noticesLength >= 1 &&
-            map(items, (bulletin, index) => (
-              <Bulletin
-                key={index}
-                items={bulletin}
-                number={bulletin[0].bulletin_number}
-                year={bulletin[0].bulletin_year}
-                date={bulletin[0].bulletin_published_on}
-              />
-            ))}
+          <div className="notice-list-table">
+            {noticesLength >= 1 &&
+              map(items, (bulletin, index) => (
+                <Bulletin
+                  key={index}
+                  items={bulletin}
+                  number={bulletin[0].bulletin_number}
+                  year={bulletin[0].bulletin_year}
+                  date={bulletin[0].bulletin_published_on}
+                />
+              ))}
+          </div>
+        </Col>
+      </Row>
+      <hr />
+      <Legend />
```

Comment:
Was there any design for this? I think the legend is too low, no one will scroll to it. And I think it's pretty important info. Couldn't it be between `Našim cieľom...` text and search box? What do you think?